### PR TITLE
Aladdin generated examples for DataFactory, RecoveryServices, Sql, Websites

### PR DESCRIPTION
**Pull request by aladdindoc**
- Add Aladdin crafted examples.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/DataFactory/DataFactoryV2/help/New-AzDataFactoryV2LinkedServiceEncryptedCredential.md [hunk 1]
```diff
@@ -32,12 +32,18 @@ The New-AzDataFactoryV2LinkedServiceEncryptCredential cmdlet encrypt credential
 ## EXAMPLES
 
 ### Example 1: Encrypt credentials in a linked service
-```
+```powershell
 PS C:\> New-AzDataFactoryV2LinkedServiceEncryptCredential -ResourceGroupName resourceGroup -DataFactoryName myDataFactory -IntegrationRuntimeName myIR -File D:\sql.json
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/DataFactory/DataFactoryV2/help/New-AzDataFactoryV2LinkedServiceEncryptedCredential.md]: Removed incorrect example

File: src/DataFactory/DataFactoryV2/help/Remove-AzDataFactoryV2IntegrationRuntimeNode.md [hunk 2]
```diff
@@ -37,12 +37,20 @@ The Remove-AzDataFactoryV2IntegrationRuntimeNode cmdlet removes a node in an int
 ## EXAMPLES
 
 ### Example 1: Remove a node from an integration runtime
-```
+```powershell
 PS C:\> Remove-AzDataFactoryV2IntegrationRuntimeNode -ResourceGroupName 'rg-test-dfv2' -DataFactoryName 'test-df-eu2' -Name 'test-selfhost-ir' -NodeName 'Node_1'
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/DataFactory/DataFactoryV2/help/Remove-AzDataFactoryV2IntegrationRuntimeNode.md]: Removed example with alias parameter.

File: src/DataFactory/DataFactoryV2/help/Sync-AzDataFactoryV2IntegrationRuntimeCredential.md [hunk 3]
```diff
@@ -37,12 +37,20 @@ The **Sync-AzDataFactoryV2IntegrationRuntimeCredential** cmdlet synchronizes on-
 ## EXAMPLES
 
 ### Example 1: Sync an integration runtime credential
-```
+```powershell
 PS C:\> Sync-AzDataFactoryV2IntegrationRuntimeCredential -ResourceGroupName 'rg-test-dfv2' -DataFactoryName 'test-df-eu2' -Name 'test-selfhost-ir'
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/DataFactory/DataFactoryV2/help/Sync-AzDataFactoryV2IntegrationRuntimeCredential.md]: Removed example with typo

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupStatus.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ If it is protected, the relevant vault details will be returned.
 ## EXAMPLES
 
 ### Example 1
-```
+```powershell
 PS C:\> $status = Get-AzRecoveryServicesBackupStatus -Name "myAzureVM" -ResourceGroupName "myAzureVMRG" -ResourceType "AzureVM"
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupStatus.md]: Removed example with typo.

File: src/RecoveryServices/RecoveryServices/help/Start-AzRecoveryServicesAsrTestFailoverCleanupJob.md [hunk 5]
```diff
@@ -36,19 +36,27 @@ The **Start-AzRecoveryServicesAsrTestFailoverCleanupJob** cmdlet starts the test
 ## EXAMPLES
 
 ### Example 1
-```
+```powershell
 PS C:\> Start-AzRecoveryServicesAsrTestFailoverCleanupJob -ReplicationProtectedItem $rpi -Comments "testing done"
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/RecoveryServices/RecoveryServices/help/Start-AzRecoveryServicesAsrTestFailoverCleanupJob.md]: Removed duplicate example with incorrect parameter name.

File: src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrProtectableItem.md [hunk 6]
```diff
@@ -24,12 +24,20 @@ The **New-AzRecoveryServicesAsrProtectableItem** adds a new protectable item to
 ## EXAMPLES
 
 ### Example 1
-```
+```powershell
 PS C:\> New-AzRecoveryServicesAsrProtectableItem -ProtectionContainer $pc -Name $name -IPAddress $ipaddresss -OSType $OsType
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrProtectableItem.md]: Removed example with incorrect parameter.

File: src/Websites/Websites/help/Update-AzWebAppAccessRestrictionConfig.md [hunk 7]
```diff
@@ -30,12 +30,18 @@ The **Update-AzWebAppAccessRestrictionConfig** cmdlet updates Access Restriction
 ## EXAMPLES
 
 ### Example 1: Update a Web App SCM Site to use Access Restrictions from Main Site
-```
+```powershell
 PS C:\>Set-AzWebAppAccessRestriction -ResourceGroupName "Default-Web-WestUS" -Name "ContosoSite" -ScmSiteUseMainSiteRestrictionConfig
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/Websites/Websites/help/Update-AzWebAppAccessRestrictionConfig.md]: Removed the example with the incorrect command.

File: src/Sql/Sql/help/Get-AzSqlSyncAgentLinkedDatabase.md [hunk 8]
```diff
@@ -23,7 +23,7 @@ The **Get-AzSqlSyncAgentLinkedDatabases** cmdlet returns information about SQL S
 ## EXAMPLES
 
 ### Example 1: Get the linked SQL Server databases for an Azure SQL sync agent.
-```
+```powershell
 PS C:\> Get-AzSqlSyncAgentLinkedDatabases -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -SyncAgentName "SyncAgent01" | Format-List
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/Sql/Sql/help/Get-AzSqlSyncAgentLinkedDatabase.md]: Removed the example with the typo.

File: src/Websites/Websites/help/Reset-AzWebAppPublishingProfile.md [hunk 9]
```diff
@@ -29,13 +29,19 @@ The **Reset-AzWebAppPublishingProfile** cmdlet resets the publishing profile for
 
 ## EXAMPLES
 
-### 1:
-```
+### Example 1
+```powershell
 PS C:\> Reset-AzWebAppSlotPublishingProfile -ResourceGroupName "Default-Web-WestUS" -Name "ContosoWebApp"
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply src/Websites/Websites/help/Reset-AzWebAppPublishingProfile.md]: Removed the example with the incorrect command.

**Comment by aladdindoc** [comment]
#sign-off

**Comment by msJinLei** [comment]
@aladdindoc I rebase your commits to the latest master and resolve a conflict. Please check whether the current codes are what you want.

**Pull request merged**